src/main/java/org/elasticsearch/common/jna/Kernel32Library.java

```diff
@@ -81,7 +124,17 @@ public boolean addConsoleCtrlHandler(ConsoleCtrlHandler handler) {
 
     interface Kernel32 extends Library {
 
-        /**
+        public boolean VirtualLock(long address, long size);
```

Comment:
This PR has become a bit stale and this class has changed a bit.  I'll be sure to document all the native methods once I update the PR.